server/data/dataAccess/db.js

```diff
@@ -14,7 +14,7 @@ const pool = new Pool({
   ssl:
     config.db.sslEnabled === 'true'
       ? {
-          ca: fs.readFileSync('root.cert'),
+          ca: fs.readFileSync('/app/root.cert'),
```

Comment:
so i can use the same db config from a new `scripts` dir outside the conventional `server` dir